### ISSUE TITLE
about the uploaded SOTS dataset

### ISSUE THREAD

**Issue by diadestiny** [IDKiro/DehazeFormer#26]
Hi, thanks for your work!
I have a simple but confusing question: the size of SOTS dataset you uploaded in Baiduyunpan link(data/RESIDE-IN/test.zip) is different from the raw official website version, I hope to know what the differences are.
download from your upload
![image](https://github.com/IDKiro/DehazeFormer/assets/44188454/758882ec-917f-4b97-aa4d-d361f3736984)
download from the raw official website 
![image](https://github.com/IDKiro/DehazeFormer/assets/44188454/1d4c1662-c05f-42a8-86cc-9a06830bd851)

the same problem with the other dataset(Reside-ITSv2) about your uploaded and raw versions.

**Comment by IDKiro** [comment]
We excluded some small images from the training set of RESIDE OUT (because the resolution of these small images is smaller than the set patch size), and the names of the excluded files are recorded in the txt file.
In addition, we use the ITS in RESIDE v0, which is consistent with most of the previous methods, as described in the paper.

**Comment by IDKiro** [comment]
I am very sorry, I found that you are asking about SOTS, I don't remember this part very well.
I should have read all the images and saved them again, in order to avoid the difference of different image codecs reading JPEG format images.
You can test it with the official test set and the results should be the same.

**Comment by diadestiny** [comment]
thanks, I will try it.
